app/views/site/welcome.html.erb

```diff
@@ -77,17 +77,20 @@
 
 <%= render "any_questions" %>
 
-<div class='text-center mb-3'>
-<% if params[:oauth_return_url] %>
-  <a class="btn btn-primary" href="<%= params[:oauth_return_url] %>"><%= t ".continue_authorization" %></a>
-<% else %>
-  <a class="btn btn-primary start-mapping" href="<%= edit_path %>"><%= t ".start_mapping" %></a>
-<% end %>
-</div>
-
 <div class='alert alert-primary'>
   <h2><%= t ".add_a_note.title" %></h2>
   <p><%= t ".add_a_note.para_1" %></p>
   <p><%= t ".add_a_note.para_2_html", :map_link => link_to(t(".add_a_note.the_map"), root_path),
                                       :note_icon => tag.span(:class => "icon note bg-dark rounded-1") %></p>
 </div>
+
+<% content_for :footer_outer_class, "sticky-bottom" %>
+<% content_for :footer do %>
+  <div class='d-flex justify-content-center align-items-center'>
+    <% if params[:oauth_return_url] %>
+      <a class="btn btn-primary d-block d-sm-inline-block" href="<%= params[:oauth_return_url] %>"><%= t ".continue_authorization" %></a>
```

Comment:
What are `d-block d-sm-inline-block` classes supposed to do on a flex item?

Reply:
To be honest, this PR was from a while ago, and I don't remember exactly why I added it. I must have encountered some layout issues at the time. However, I've now removed it locally, and everything looks fine on both small and large screens, so I'll just go ahead and remove it since it seems unnecessary.

Reply:
Updated the code, thank you.